configuration/defaults/mainnet/configuration.yaml

```diff
@@ -220,21 +222,27 @@ options:
   # and override and not an extension so anything matched here will not
   # go to the default backend, only to the explicitly listed backends.
   mapBackends:
+    cardano.node.BlockFetchDecision.peers:
+      - TraceForwarderBK
+      - EKGViewBK
+      - kind: UserDefinedBK
+        name: LiveViewBackend
```

Comment:
We have no setup above for a `UserDefinedBK`. How does this interact with the `setupBackends` directive?

Reply:
this is the LiveView backend that will be setup in the code depending on the switch in the configuration.